### PR TITLE
Ivan model tests

### PR DESCRIPTION
**Pull request by ivangit546**

### CODE REVIEW

File: .venv/lib/python3.12/site-packages/asgiref/py.typed
Comment (danzzu) [inline .venv/lib/python3.12/site-packages/asgiref/py.typed:1]: Please revert this Virtual Environment file back.

**Review by danzzu — CHANGES_REQUESTED** [review]

File: .venv/lib/python3.12/site-packages/asgiref/py.typed
Comment (danzzu) [reply .venv/lib/python3.12/site-packages/asgiref/py.typed:1]: File had been reverted

**Review by danzzu — APPROVED** [review]

**Review by flickin4 — APPROVED** [review]
Nice job 👍👍👍💯

**Review by DeHampster — APPROVED** [review]
looks great

**Pull request merged**